tests/e2e/UsingGKE.md

```diff
@@ -62,6 +62,7 @@ kubectl create clusterrolebinding myname-cluster-admin-binding  --clusterrole=cl
 cat .istiorc.mk
 HUB=costinm
 TAG=mybranch
+GS_BUCKET=mybucket
```

Comment:
why this?

Reply:
I have updated the comment to hopefully make it clearer. If not set, the GS_BUCKET *istio-artifacts* will be used (I think for the istioctl) and I do not have write permissions to that bucket.